httpx/_api.py

```diff
@@ -226,9 +231,7 @@ def head(
     **Parameters**: See `httpx.request`.
 
     Note that the `data`, `files`, and `json` parameters are not available on
-    this function, as `HEAD` requests should not include a request body. The
-    `HEAD` method also differs from the other cases in that `allow_redirects`
-    defaults to `False`.
+    this function, as `HEAD` requests should not include a request body.
```

Comment:
Should this be submitted as a separate cleanup PR?

Reply:
Sure 💯